examples/flows/standard/flow-with-package-tool-using-custom-strong-type-connection/flow.dag.yaml

```diff
@@ -0,0 +1,18 @@
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
```

Comment:
although this is a flow, but I think maybe put it under the tools folder, as the import topic here is the tools

Reply:
moved to tool folder